packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -207,7 +207,11 @@ class WKScriptMessageData {
 ///
 /// See https://developer.apple.com/documentation/foundation/nshttpcookie?language=objc.
 class NSHttpCookieData {
-  late Map<NSHttpCookiePropertyKeyEnumData?, String?> properties;
+  // `NSDictionary`s are unable to use data classes as keys because they don't
+  // conform to `NSCopying`. This splits the map of properties into a list of
```

Comment:
Could you file an issue about this and link to it from the comment? It seems like something we could add support for.

Reply:
Done: https://github.com/flutter/flutter/issues/103383